cleverhans/model_abs.py

```diff
@@ -0,0 +1,73 @@
+from abc import ABCMeta
+from keras.models import Model
```

Comment:
OK, I see why this is needed for `KerasModelWrapper`. I think either `KerasModelWrapper` should go in a separate module that imports `keras`, or `KerasModelWrapper` should import `keras` only when the `KerasModelWrapper` class is constructed. Users shouldn't need to install `keras` to be able to import `ModelAbstraction`.